### PR TITLE
Fix #317 and #318: Rating form fixes

### LINKED ISSUE #317: hide rating form for app owners
Currently app owners also see app rating forms for their own apps. While it's statistically irrelevant people seem to think that it's a way to cheat the system. We should therefore simply hide the form and button for owners.

### PR DESCRIPTION
**Pull request by janibonnevier**
Fix #317 and #318 

- Explain reviews
- Button says 'Update rating' if rating already exists.
- Hide rating form from app owner
- Update mailmap and authors

### CODE REVIEW

File: nextcloudappstore/core/templates/app/detail.html [hunk 1]
```diff
@@ -119,23 +119,28 @@ <h4 class="section-heading" id="comments">{% trans 'Discussion' %}</h4>
...
         <div class="row app-ratings {% if not user_has_rated_app %}collapse{% endif %}" id="app-ratings">
             <div class="col-md-12">
                 <form action="{% url 'app-detail' object.id %}" method="post">
                     {% csrf_token %}
                     {% include 'form-fields.html' with form=rating_form %}
                     <p class="help-block">
                         {% blocktrans %}Reviews support <a rel="noopener noreferrer" href="https://daringfireball.net/projects/markdown/syntax">Markdown</a>.
-                            Do not use reviews to report bugs or request features. Neither developers nor users will be notified by your review.{% endblocktrans %}
+                            Do not use reviews to report bugs or request features. Neither developers nor users will be notified by your review.
+                            A rating without a review still counts, but will not be listed below.{% endblocktrans %}
```
Comment (BernhardPosselt) [inline nextcloudappstore/core/templates/app/detail.html]: A rating without comment (check the exact label)

Comment (BernhardPosselt) [inline nextcloudappstore/core/templates/app/detail.html]: OK it's named "rating comment"

**Comment by BernhardPosselt** [comment]
Only the help string should be adjusted to use the exact field label, otherwise looks good

**Comment by janibonnevier** [comment]
@BernhardPosselt, is there anything here that should be mentioned in the changelog?

**Comment by BernhardPosselt** [comment]
👍

**Pull request merged**